vagrant/provisioning/arkcase-ce-facts.yml

```diff
@@ -430,4 +430,17 @@ confluent_major_version: 5.4
 confluent_minor_version: 7
 confluent_scala_version: 2.12
 
-ldap_type: samba
\ No newline at end of file
+ldap_type: samba
+
+# OIDC authentication properties
+#oidc_registrationId: ""
+#oidc_clientId: ""
+#oidc_clientSecret: ""
+#oidc_redirectUri: ""
+#oidc_authorizationUri: ""
+#oidc_tokenUri: ""
+#oidc_jwkSetUri: ""
```

Comment:
"jwkSetUri", what is "jwk"?

Reply:
URI for the JSON Web Key (JWK) Set endpoint

Reply:
The naming comes from Spring. Not sure how they end up with **jwkSetUri** :).